app/components/gh-member-avatar.js

```diff
@@ -21,7 +21,7 @@ export default Component.extend({
     }),
 
     backgroundStyle: computed('member.{name,email}', function () {
-        let name = this.member.name || this.member.email;
+        let name = this.member.name || this.member.email || 'NM';
```

Comment:
@peterzimon came up with this 'NM' piece as a placeholder for New Member initials, without this the screen looks broken. It changes to normal initials calculation once email or name is entered. Lmk if you'd like to put something different here ;)